src/test/scala/chiseltest/experimental/tests/VerilatorCoverageTests.scala

```diff
@@ -7,79 +7,64 @@ import java.io.{ByteArrayOutputStream, File, PrintStream}
 import chisel3._
 import chiseltest._
 import chiseltest.stage._
+import firrtl.AnnotationSeq
+import firrtl.options.TargetDirAnnotation
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
 
 class VerilatorCoverageTests extends AnyFlatSpec with ChiselScalatestTester with Matchers {
   behavior of "Testers2"
 
-  it should "allow specifying toggle coverage for Verilator" in {
-    val coverageName = "test_run_dir/Testers2_should_allow_specifying_toggle_coverage_for_Verilator/logs/coverage.dat"
-    val coverage = new File(coverageName)
+  private def coverageTest(annotations: AnnotationSeq) = {
     val outputStream = new ByteArrayOutputStream()
-    Console.withOut(new PrintStream(outputStream)) {
-      test(new MultiIOModule {}).withAnnotations(Seq(VerilatorBackendAnnotation, ToggleCoverageAnnotation)) { c => }
+    val targetDir = Console.withOut(new PrintStream(outputStream)) {
+      test(new MultiIOModule {}).withAnnotations(annotations) { c => }.collectFirst{
+        case TargetDirAnnotation(t) => t
+      }.get
     }
-    val output = outputStream.toString
+    val coverage = new File(targetDir + s"${java.io.File.separator}logs${java.io.File.separator}coverage.dat")
     coverage.exists() should be(true)
-    output.contains("--coverage-toggle") should be(true)
-    output.contains("--coverage-line") should be(false)
-    output.contains("--coverage-user") should be(false)
+    outputStream.toString
+  }
+
+  it should "allow specifying toggle coverage for Verilator" in {
```

Comment:
Instead of commenting out tests they should be marked as ignored.

Reply:
will fix.